src/test/scala/cromwell/util/SampleWdl.scala

```diff
@@ -1668,4 +1668,32 @@ object SampleWdl {
       """.stripMargin
     override val rawInputs = Map.empty[String, String]
   }
+
+  /**
+    * Outputs which reference themselves.
+    */
+  object InputsAndOutputsAutoReferentiation extends SampleWdl {
```

Comment:
Yes, an auto-referencing output actually should be an error.  Are these changes robust to that?